oneview/resource_hypervisor_manager.go

```diff
@@ -0,0 +1,395 @@
+// (C) Copyright 2016 Hewlett Packard Enterprise Development LP
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// You may not use this file except in compliance with the License.
+// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+
+package oneview
+
+import (
+	"encoding/json"
+	"github.com/HewlettPackard/oneview-golang/ov"
+	"github.com/HewlettPackard/oneview-golang/utils"
+	"github.com/hashicorp/terraform/helper/schema"
+	"io/ioutil"
+)
+
+func resourceHypervisorManager() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceHypervisorManagerCreate,
+		Read:   resourceHypervisorManagerRead,
+		Update: resourceHypervisorManagerUpdate,
+		Delete: resourceHypervisorManagerDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"available_dvs_versions": {
+				Type:     schema.TypeSet,
+				Computed: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+
+			"category": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"created": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"description": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+
+			"display_name": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+
+			"etag": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"hypervisor_type": {
```

Comment:
As per REST API documentation, hypervisorType should be optional not Computed.

Reply:
Done. Changed it to Optional